### DISCUSSION TITLE
Unit tests

### DISCUSSION THREAD

**Discussion by sestegra** [vmware/packer-examples-for-vsphere#73] [Ideas]
Hello,

WIth #71 and #72, the unit tests would be easier to implement.

I suggest to use `govc` to create a VM from the created template and check if running and contains what is expected with [testinfra](https://testinfra.readthedocs.io/en/latest/) for instance.

What do you think about it?

Kind regards,
Stéphane

**Comment by tenthirtyam** [comment]
Tracking under https://github.com/vmware-samples/packer-examples-for-vsphere/issues/264 for project enhancement.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.
